dotnet/CONTRIBUTING.md

```diff
@@ -14,3 +14,9 @@ Alternatively, run `msbuild` from this directory.
 
 Keep in mind that this will only run unit tests. The acceptance tests are only
 run when you build with `make`.
+
+## Make a release
+
+
+    # Change version in `Gherkin.NuGetPackages\Gherkin.nuspec`
+    .nuget\NuGet.exe pack Gherkin.NuGetPackages\Gherkin.nuspec
```

Comment:
Stupid question regarding "there is no `Gherkin.dll` file in `bin`": did you compile the solution before you ran NuGet.exe pack? The compilation process drops the `Gherkin.dll` file in the `bin` directory - so it's probably a good idea to mention that step as well.

I don't know whether forward slashes in a `.nuspec` file are OK, I will investigate that.